Sofa/Component/Topology/Mapping/src/sofa/component/topology/mapping/Edge2QuadTopologicalMapping.h

```diff
@@ -31,6 +31,13 @@
 #include <sofa/core/BaseMapping.h>
 #include <sofa/core/behavior/MechanicalState.h>
 
+
+namespace sofa::component::topology::container::dynamic
```

Comment:
I am surprised that forward declaration works with Links

Reply:
I wanted to try for the lolz but it works on MSVC. Curious to see if it will be the same for gcc/clang (if the CI does want to do its job) 😅